MeNext.Spotify.Droid/PlayerListener.cs

```diff
@@ -111,24 +118,28 @@ void OnAuthenticationComplete(AuthenticationResponse response)
 
         // == Callback methods == //
 
+        //Update on login
```

Comment:
Use `///` comments for method headers. When using `//` comments, you should have a space before text.